### PR TITLE
Fetch a facebook page id

### PR DESCRIPTION
**Pull request by bruce-fs**
- as part of Funky::Video API

### CODE REVIEW

File: lib/funky/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Funky
-  VERSION = "0.2.9"
+  VERSION = "0.3.0"
```
Comment (claudiofullscreen) [inline lib/funky/version.rb]: @bruce-fs Backward-compatible changes only bump the patch version when version < 1.

So this should be `"0.2.10"`

File: spec/videos/video_spec.rb [hunk 2]
```diff
@@ -19,6 +19,7 @@
       it { expect(video.length).to be_a(Float) }
       it { expect(video.picture).to be_a(String)}
       it { expect(video.page_name).to be_a(String) }
+      it { expect(video.page_id).to be_a(String) }
```
Comment (claudiofullscreen) [inline spec/videos/video_spec.rb]: @bruce-fs Does _every_ Facebook video return a value for this field?

Or some videos return `nil`?

**Comment by bruce-fs** [comment]
@claudiofullscreen, some videos will likely return nil I believe

File: spec/videos/video_spec.rb (hunk 2 above)
Comment (bruce-fs) [inline spec/videos/video_spec.rb]: @claudiofullscreen, some videos will likely return nil I believe

Comment (bruce-fs) [inline spec/videos/video_spec.rb]: @claudiofullscreen - this method is analagous to page_name, though - if you use the Funky::Video find/where methods of the API, you will get `Funky::ContentNotFound` or an `[]`

File: lib/funky/video.rb [hunk 3]
```diff
@@ -45,6 +45,15 @@ def page_name
       data.fetch(:from)[:name]
     end
 
+    # @return [String] the id of Facebook page for the video.
+    def page_id
+      data.fetch(:from)[:id]
+    end
+
+    def page_url
+      "https://www.facebook.com/" + page_id
```
Comment (claudiofullscreen) [inline lib/funky/video.rb]: @bruce-fs  Interpolation might be better performing than concatenation.

See https://github.com/johnotander/ruby_optimization_techniques#45-simple-code-enhancements

Comment (claudiofullscreen) [inline lib/funky/video.rb]: Other than that, this PR looks good to me.

**Pull request merged**